### PR TITLE
Create Recap of First Black Python Devs in Person Meetup #382

### PR DESCRIPTION
**Pull request by AfiMaameDufie**
## Issue Link 🔗: https://github.com/BlackPythonDevs/blackpythondevs.github.io/issues/382

**Issue**: #[Issue_Number] 382

## Type of Change

- [ ] Bug fix 🐞
- [ ] New feature/page
- [x] Documentation update
- [ ] Other

## Description 📋

Added a blog for the BPD Ghana Connect Meetup Recap

## Checklist ✅

- [x] Followed the [Code of Conduct](https://github.com/BlackPythonDevs/blackpythondevs.github.io?tab=coc-ov-file) and [Contribution Guide](https://github.com/BlackPythonDevs/blackpythondevs.github.io/blob/469afd71943738b7209bfe56d5a4e35394fbaafc/CONTRIBUTING.md)
- [x] Ran `pre-commit run --all`
- [x] All tests pass locally
- [ ] Added tests (if applicable)
- [x] Documentation updated (if applicable)

## Additional Notes & Screenshots

Screenshot 2024-10-13 at 3 29 48 PM

Add any additional notes or comments that might be helpful for the reviewers.

### CODE REVIEW

**Review by TimOsahenru — COMMENTED** [review]
LGTM! Just a little suggestion—can we include a featured image as well? It's fine if none is available, something like this image. Thanks

**Comment by kjaymiller** [comment]
@AfiMaameDufie - LGTM I agree with @TimOsahenru. A featured image would be great to add.

**Comment by AfiMaameDufie** [comment]
@TimOsahenru , @kjaymiller 

In my commits, I removed large images since the checks were failing. I found it hard to compress the pictures; I tried a couple of times for the feature image.

I will try again at the end of the day and revert.

**Comment by kjaymiller** [comment]
> @TimOsahenru , @kjaymiller
> 
> In my commits, I removed large images since the checks were failing. I found it hard to compress the pictures; I tried a couple of times for the feature image.
> 
> I will try again at the end of the day and revert.

I understand this issue. 

One suggestion would be to convert the file to webp (I can help you with that) and compress it down to the maximum size it needs to be (likly 1920x1080 or smaller).

**Comment by AfiMaameDufie** [comment]
> > @TimOsahenru , @kjaymiller
> > In my commits, I removed large images since the checks were failing. I found it hard to compress the pictures; I tried a couple of times for the feature image.
> > I will try again at the end of the day and revert.
> 
> I understand this issue.
> 
> One suggestion would be to convert the file to webp (I can help you with that) and compress it down to the maximum size it needs to be (likly 1920x1080 or smaller).

ohhh, webp, thanks! Will do that and push again.

**Comment by TimOsahenru** [comment]
@AfiMaameDufie not sure if this helps, but you can also try using this online tool https://imageresizer.com/ to resize the image.

**Comment by AfiMaameDufie** [comment]
Pushed again @TimOsahenru 
Should I take out the image in the blog and just leave the feature image?

**Comment by TimOsahenru** [comment]
@AfiMaameDufie I think we have the provision to display the other images you've included, it's fine to leave them. Thanks

**Comment by AfiMaameDufie** [comment]
> @AfiMaameDufie I think we have the provision to display the other images you've included, it's fine to leave them. Thanks
Great, thank you.

File: _posts/2024-10-13-black-python-devs-ghana-event.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+![Black Python Devs Ghana Connect Meetup](/assets/images/bpd3.webp)
```
Comment (kjaymiller) [inline _posts/2024-10-13-black-python-devs-ghana-event.md]:
```suggestion
![Black Python Devs Ghana Connect Meetup](/assets/images/bpd3.webp)

```

Comment (kjaymiller) [reply _posts/2024-10-13-black-python-devs-ghana-event.md]: Unless the linter complainsI would include a space to get things on separate lines.

File: assets/images/bpd1.webp
Comment (kjaymiller) [inline assets/images/bpd1.webp:1]: I would name this something a little more specific like bpd-ghana-first-meetup-1.webp

File: assets/images/bpd3.webp
Comment (kjaymiller) [inline assets/images/bpd3.webp:1]: I would name this something a little more specific like bpd-ghana-first-meetup-3.webp (what happened to two?)

**Review by kjaymiller — CHANGES_REQUESTED** [review]
@AfiMaameDufie - great work on this.

My changes are minor nitpicks (naming and some formatting)

**Comment by kjaymiller** [comment]
@AfiMaameDufie were you able to rename the files to something that would be more easily identifiable.

File: _posts/2024-10-13-black-python-devs-ghana-event.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
```
Comment (kjaymiller) [inline _posts/2024-10-13-black-python-devs-ghana-event.md]: @AfiMaameDufie don't forget to rename the files in the post as well

**Comment by AfiMaameDufie** [comment]
@kjaymiller 

Final review please

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**